src/server/fmd/origin/section.test.js

```diff
@@ -12,15 +14,32 @@ describe('OriginSection', () => {
     expect(OriginSection.config.summaryLink).toBe(checkAnswersPage.urlPath)
   })
 
-  it('should have the correct first page', () => {
-    expect(OriginSection.firstPageFactory()).toBeInstanceOf(MockOriginPage)
+  it.each([
+    ['milk', PremisesTypePage],
+    ['not-milk', TlaPage]
+  ])('should have the correct start page', (value, expectedPage) => {
+    expect(
+      OriginSection.firstPageFactory({
+        about: {
+          whatIsMoving: value
+        }
+      })
+    ).toBeInstanceOf(expectedPage)
   })
 
-  it('should not be visible', () => {
-    expect(OriginSection.config.isVisible({})).toBe(false)
+  it('should be visible', () => {
+    expect(OriginSection.config.isVisible({})).toBe(true)
   })
 
   it('should not be enabled', () => {
     expect(OriginSection.config.isEnabled({})).toBe(false)
   })
+
+  it('should be enabled', () => {
```

Comment:
"should be enabled when the about section is complete", and similar comment for the other test for "not enabled"